Symfony/CS/Fixer/Symfony/UnusedUseFixer.php

```diff
@@ -38,6 +38,36 @@ public function fix(\SplFileInfo $file, $content)
         return $tokens->generateCode();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // should be run after the MultipleUseFixer
+        return -10;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(\SplFileInfo $file)
+    {
+        // some fixtures are auto-generated by Symfony and may contain unused use statements
+        if (false !== strpos($file, DIRECTORY_SEPARATOR.'Fixtures'.DIRECTORY_SEPARATOR)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
```

Comment:
oh, right. well, i don't have time to mess about with that now, and i also disagree. having them in the order you call them makes more sense to me.